projects/kit/pipes/extract-country-code/extract-country-code.module.ts

```diff
@@ -0,0 +1,12 @@
+import {NgModule} from '@angular/core';
+import {TuiIsoToCountryCodeModule} from '@taiga-ui/kit/pipes/iso-to-country-code';
+
+import {TuiExtractCountryCodePipe} from './extract-country-code.pipe';
+
+@NgModule({
+    declarations: [TuiExtractCountryCodePipe],
+    exports: [TuiExtractCountryCodePipe],
+    providers: [TuiExtractCountryCodePipe],
+    imports: [TuiIsoToCountryCodeModule],
```

Comment:
I don't like that we are creating convoluted DI solution. How about we create a pure utility function to perform the action and use it in both pipes, instead of adding pipes to providers and injecting them?